pkg/api/tenant.go

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package api
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/grafana/dskit/middleware"
+	"github.com/grafana/dskit/tenant"
+)
+
+const (
+	tooManyTenantsTemplate = "too many tenant IDs present. max: %d actual: %d"
```

Comment:
[nit] WDYT about further clarify it?

```suggestion
	tooManyTenantsTemplate = "too many tenant IDs present in the request. max: %d actual: %d"
```